design-reviews/recipes/feature-story-design-review-template.md

```diff
@@ -0,0 +1,60 @@
+# Your Feature or Story Design Title Here
+
+> Does the feature re-use or extend existing patterns / interfaces that have already been established for the project?
+> Does the feature expose new patterns or interfaces that will establish a new standard for new future development?
+
+* [Feature/Story Name](http://link-to-feature-or-story-work-item)
+* Engagement: [Engagement]
+* Customer: [Customer]
+* Authors: [Author1, Author2, etc.]
+
+## Overview/Problem Statement
+
+* Describe the feature/story with a high-level summary.
```

Comment:
Maybe similar to what you did in the task template add a 1st bullet: "It can also be a link to the work item"?

Reply:
+1 - I would also suggest adding a link to the project or design doc for the project that covers this, to give more context onto what is being done

Reply:
Added.